head_test.go

```diff
@@ -434,15 +426,16 @@ Outer:
 			} else {
 				testutil.Equals(t, 1, len(lns))
 				testutil.Equals(t, 1, len(lvs))
-				testutil.Equals(t, "a", lns[0])
-				testutil.Equals(t, "b", lvs[0])
+				testutil.Equals(t, lblDefault.Name, lns[0])
+				testutil.Equals(t, lblDefault.Value, lvs[0])
 			}
 
 			for {
 				eok, rok := expSeriesSet.Next(), actSeriesSet.Next()
 				testutil.Equals(t, eok, rok)
 
 				if !eok {
+					testutil.Assert(t, rok == false, "actual series set include more series than expected")
```

Comment:
This already gets tested above in `testutil.Equals(t, eok, rok)` for equality. So this would never be false.

Reply:
yes, updated